stdlib/public/core/Set.swift

```diff
@@ -1223,7 +1223,11 @@ extension Set {
   @inlinable
   public __consuming func intersection(_ other: Set<Element>) -> Set<Element> {
     var newSet = Set<Element>()
-    for member in self {
+    var (iter, other) = (self, other)
```

Comment:
Just one nit: I’m not sure I understand why the variable is called `iter`: it’s not an iterator, and it’s either self or other. Why don’t we just call it `smaller` and `larger` or just `set1` and `set2`?

Reply:
Yes, I do think that naming makes sense; variables named `smaller` and `larger` does make it easier to read.

```swift
let (smaller, larger) =
  count < other.count ? (self, other) : (other, self)
for member in smaller {
  if larger.contains(member) {
    newSet.insert(member)
  }
}
```
 That should work right @xwu? If so, let me go ahead and make that change.

Reply:
@BenziAhamed Sounds reasonable to me!

Reply:
Done @xwu. Please review.